src/blocks/ImageCard.scss

```diff
@@ -185,3 +192,13 @@
     box-shadow: 0 0 3px 4px var(--bloom-color-accent-cool);
   }
 }
+
+.image-card__inner-modal {
```

Comment:
I'd be curious on how we could refactor this piece since I generally try to avoid using !important in CSS. From first glance, I notice that we can override some of the Modal css variables (ie --content-padding-block).

Reply:
Nice! Thanks for updating that.